requirements.txt

```diff
@@ -2,3 +2,4 @@ requests==2.25.1
 gate_api==4.22.2
 PyYAML==6.0
 pytest==6.2.5
+pytest-cov==3.0
```

Comment:
Someone added code coverage to the project, but forgot to add the dependency to `requirements.txt`.